_templates/exchanges.html

```diff
@@ -223,9 +226,17 @@ <h3 id="south-africa" class="no_toc">South Africa</h3>
         </p>
       </div>
     </div>
-
-  </div>
-</div>
+    <div class="row marketplace">
+      <img class="marketplace-flag" src="/img/flags/UG.svg?{{site.time | date: '%s'}}" alt="Ugandan flag">
+      <div>
+        <h3 id="Uganda" class="no_toc">Uganda</h3>
+        <p>
+         <a class="marketplace-link" href="https://binance.co.ug">Binance Uganda</a>
```

Comment:
This Ugandan Binance website no longer appears to be loading. It's timing out when we try to visit it.

Reply:
@wbnns the website loads, the url is just incorrect. 

https://www.binance.co.ug should work

Reply:
@JagerGizmo Ok, great, thanks.

@tbsd Could you please update the Ugandan link with the correct address?